x-pack/plugins/actions/server/routes/find.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import {
+  IRouter,
+  RequestHandlerContext,
+  KibanaRequest,
+  IKibanaResponse,
+  KibanaResponseFactory,
+} from 'kibana/server';
+import { FindOptions } from '../../../../legacy/plugins/alerting/server/alerts_client';
+import { LicenseState } from '../lib/license_state';
+import { verifyApiAccess } from '../lib/license_api_access';
+
+// config definition
+const querySchema = schema.object({
+  per_page: schema.number({ defaultValue: 20, min: 0 }),
+  page: schema.number({ defaultValue: 1, min: 1 }),
+  search: schema.maybe(schema.string()),
+  default_search_operator: schema.oneOf([schema.literal('OR'), schema.literal('AND')], {
+    defaultValue: 'OR',
+  }),
+  search_fields: schema.maybe(schema.oneOf([schema.arrayOf(schema.string()), schema.string()])),
+  sort_field: schema.maybe(schema.string()),
+  has_reference: schema.maybe(
```

Comment:
do we even need the `maybe()` here, on a `nullable()`?  I guess we want `maybe()` long-term, the `nullable()` use is until it's fixed?

Reply:
nullable lets you specify null in the type, which is another _case_ to support.
Better to either have the key with a value, or not have the key at all, IMHO.